Configuration/Skimming/python/PDWG_EXOSoftDisplacedVertices_cff.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.EventContent.EventContent_cff import AODSIMEventContent
+EXOSoftDisplacedVerticesSkimContent = AODSIMEventContent.clone()
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi as _hltHighLevel
+
+hltSoftDV = _hltHighLevel.hltHighLevel.clone(
+   throw = False,
+   andOr = True,
+   HLTPaths = [
+    "HLT_PFHT*_PFMET*_PFMHT*_IDTight_v*",
+    "HLT_PFMET*_PFMHT*_IDTight_v*",
+    "HLT_PFMET*_PFMHT*_IDTight_PFHT*_v*",
+    "HLT_PFMETTypeOne*_PFMHT*_IDTight_v*",
+    "HLT_PFMETTypeOne*_PFMHT*_IDTight_PFHT*_v*",
+    "HLT_PFMETNoMu*_PFMHTNoMu*_IDTight_v*",
+    "HLT_PFMETNoMu*_PFMHTNoMu*_IDTight_PFHT*_v*",
+    "HLT_MonoCentralPFJet*_PFMETNoMu*_PFMHTNoMu*_IDTight_v*",
+    "HLT_PFMET*_*Cleaned_v*"
+   ]
+)
+
+softDVSelection=cms.EDFilter("CandViewSelector",
+    src = cms.InputTag("pfMet"),
+    cut = cms.string( "pt()>140" ),
+    filter = cms.bool(True)
+)
+
+EXOSoftDisplacedVerticesSkimSequence = cms.Sequence(
+    hltSoftDV * softDVSelection
```

Comment:
out of curiosity, what's "soft displaced vertices" specific here? It looks more like a skim on the MET values.

Reply:
The skim is for the Soft Displaced Vertices plus MET analysis. Its purpose is to access tracks below the MiniAOD threshold (~1GeV) in order to reconstruct very soft displaced vertices in addition to MET requirements.

Reply:
For my education, is it customary to call the SKIM by the target analysis rather than the actual skimming pattern? I was wondering if this could be useful to other analyses that might get discouraged by the very specific analysis targeted nomenclature.

Reply:
This was our understanding, given the different existing skims. But we are open to modify the name if there is a good suggestion. We would just like to avoid a confusion with PDWG_EXOHighMET.

Reply:
> But we are open to modify the name if there is a good suggestion.

I am also genuinely curious about the policy, that's why I asked.

Reply:
Checking the other SKIMs I can't say we really have a strict policy for SKIM definition (it's either the analysis either the  content). In this case maybe it's a bit odd since the "analysis name" could also be referring to a sort of vertex selection that actually is not there. But I can't think of a quick good alternative (at the moment).

Reply:
SKIM is usually for special tasks or analyses in POG/PAG (instead of general purpose) since these datasets (RECO, AOD) could not be on disk. Different SKIMs also focus on different data tiers. Therefore,  for my understanding, there is no clear policy or role for the name of SKIM but depending on the SKIM proposers to label the purpose of the SKIM. So, the proposers can quickly recognize which datasets are what they want. 

I’m not sure if other analyses aren’t relevant to the SKIM purpose but would like to use the same SKIMs. Of course, in case the new proposed SKIM has highly similar event contents and filers, we can suggest adding additional requirements to the existing SKIM config instead of creating the new one.